### ISSUE TITLE
About fixing the bias in the layers

### ISSUE THREAD

**Issue by b224618** [RAIVNLab/supsup#22]
Hi, 

Thanks a lot for sharing the code. I have a small question. It seems that the bias of the layers are not masked by the score and not freezed either, and I'm confused about this. Is the bias term different from other trainable parameters (weights)?

Thanks!

**Comment by mitchellnw** [comment]
Hello,

Thanks for the question! Do you mean in the convs? We are always setting `bias=False` in the convs (e.g., https://github.com/RAIVNLab/supsup/blob/master/models/builder.py#L29-L66).

**Comment by b224618** [comment]
Hi,

Thanks for the quick reply! Now I understand, I have misunderstood the code. By the way, I'm still  little bit confused why can't the bias be masked in the same way as the weights?

**Comment by mitchellnw** [comment]
Yes, it could! I don't know if that would make much of a difference.

**Comment by b224618** [comment]
Got it! Thanks a lot for your help!
